goaviatrix/edge_neo_device_onbarding.go

```diff
@@ -19,12 +19,12 @@ type EdgeNEODevice struct {
 }
 
 type EdgeNEODeviceNetwork struct {
```

Comment:
I note its not your PR, but we can fix the name of this file.  First off its say `onbarding` second we don't really want to use the "neo" term.  Can we call this `edge_platform_device_onboarding.go` ? or something like that?